packages/p2p-v2/src/components/FormProgress/FormProgress.scss

```
@@ -0,0 +1,77 @@
+.p2p-v2-form-progress {
+    width: 100%;
+    position: relative;
+
+    &__header {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-bottom: 3.2rem;
+
+        & > h2 {
+            text-transform: none;
+            margin: 2rem 0 0;
+        }
+    }
+    &__step {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        height: 6rem;
+        justify-content: space-around;
+        width: 160px;
+        z-index: 2;
+
+        & .p2p-v2-identifier {
+            border-radius: 50%;
+            display: block;
+            width: 24px;
+            background-color: #999999;
+            border: 1px solid #999999;
+
+            &--active {
+                background-color: #ff444f;
+                border: 1px solid #ff444f;
+            }
+        }
+    }
+    &__steps {
+        align-items: center;
+        display: flex;
+        justify-content: center;
+        margin-top: 20px;
+        position: relative;
+
+        &--before {
+            bottom: 0;
+            content: '';
+            left: 0;
+            margin: 0 auto; /* this centers the line to the full width specified */
+            position: absolute; /* positioning must be absolute here, and relative positioning must be applied to the parent */
+            right: 0;
+            top: 1.7rem;
+            border-top: 2px solid #999999;
+        }
+        &--after {
+            border-top: 2px solid #ff444f;
+            transition: width 0.3s ease;
+            @include desktop {
+                bottom: 0;
+                content: '';
+                left: 0;
+                margin: 0 auto; /* this centers the line to the full width specified */
+                position: absolute; /* positioning must be absolute here, and relative positioning must be applied to the parent */
+                top: 1.7rem;
+            }
+            @include mobile {
+                position: absolute;
+                top: 0;
+            }
+        }
```

since both desktop and mobile use position: absolute, can we do this?
```suggestion
            border-top: 2px solid #ff444f;
            transition: width 0.3s ease;
            position: absolute;
            
            @include desktop {
                bottom: 0;
                content: '';
                left: 0;
                margin: 0 auto; /* this centers the line to the full width specified */
                top: 1.7rem;
            }
            @include mobile {
                top: 0;
            }
        }
```